pkg/patch/multi.go

```diff
@@ -178,7 +178,7 @@ func patchMultiPlatformImage(
 					Platform: platformKey,
 					Status:   "Not Patched",
 					Ref:      originalRef.String() + " (original reference)",
-					Message:  "Preserved original image (No Scan Report provided for platform)",
+					Message:  "Preserved original image (Platform not provided via --platform)",
```

Comment:
This will not always be the case. If the scan report dir is provided, the reasoning in the message should be "no scan report" for that platform. However, if it is filtered via the --platform flag it should be this.